tests/integration/PhotoDeletion.test.js

```diff
@@ -164,10 +165,11 @@ describe( "Photo Deletion", ( ) => {
     await confirmPhotosAndAddTopId();
     await saveAndEditObs();
     // Enter camera to add new photo
-    const addEvidenceButton = await screen.findByLabelText( "Add evidence" );
-    await actor.press( addEvidenceButton );
-    const addEvidenceSheet = await screen.findByTestId( "AddEvidenceSheet" );
-    const cameraButton = await within( addEvidenceSheet ).findByLabelText( "Camera" );
+    const addEvidenceButton = await waitFor( () => screen.findByLabelText( "Add evidence" ) );
```

Comment:
I think all of the findBy* helpers basically do `waitFor` internally, so this should not be necessary, especially if you're not manually specifying a timeout.